static/src/javascripts/projects/common/views/commercial/creatives/fabric.html

```diff
@@ -0,0 +1,17 @@
+<div
+    class="fabric"
+    style="
+        background-image: url(<%=backgroundImage%>);
+        background-color: <%=backgroundColour%>;
+        background-repeat: <%=backgroundRepeat%>;
+        background-position: <%=backgroundPosition%>;
+        background-attachment: <%=(backgroundScroll === 'reveal' ? 'fixed' : 'scroll')%>;
```

Comment:
I saw it when I tried to precompile lodash templates, and yet again when we worked on gimbap with @uplne. I think it makes better sense to treat lodash templates as dumb vessels for values.